src/components/patientPage/patientCard/index.tsx

```diff
@@ -16,42 +16,27 @@ const PatientCard = ({ patient }: Props) => {
   getPatientNumberCard and getSubjectNumberCard are now rendering PatientInfo elements with click option which print the results on the console.
   */
   const getPatientNumberCard = (
-    object: "observations" | "conditions",
+    object:
+      | "observations"
+      | "conditions"
+      | "allergyIntolerances"
+      | "episodesOfCare",
     writtenName: string
   ) => {
+    let resourceNumber: string = "";
     if (patient[object]) {
-      return (
-        <div
-          onClick={() => {
-            console.log(writtenName + " : ", patient[object].entry);
-          }}
-        >
-          <PatientGeneralInfo
-            type={writtenName}
-            content={patient[object].total.toString()}
-          />
-        </div>
-      );
-    }
-  };
+      resourceNumber = patient[object].length;
+    } else resourceNumber = "0";
 
-  const getSubjectNumberCard = (
-    object: "allergyIntolerances" | "episodesOfCare",
-    writtenName: string
-  ) => {
-    if (patient[object])
-      return (
-        <div
-          onClick={() => {
-            console.log(writtenName + " : ", patient[object].entry);
-          }}
-        >
-          <PatientGeneralInfo
-            type={writtenName}
-            content={patient[object].total.toString()}
-          />
-        </div>
-      );
+    return (
+      <div
+        onClick={() => {
+          console.log(writtenName + " : ", patient[object]);
+        }}
```

Comment:
I guess it was used for debug purpose. Maybe you can remove it now

Reply:
done